lib/Runtime/Language/i386/AsmJsJitTemplate.cpp

```diff
@@ -551,14 +551,13 @@ namespace Js
         AsmJsSIMDValue* simdArg;
 
         // setup stack memory
-        FrameDisplay* frame = func->GetEnvironment();
-        Var moduleEnv = frame->GetItem(0);
-        Var* arrayBufferVar = (Var*)frame->GetItem(0) + AsmJsModuleMemory::MemoryTableBeginOffset;
+        AsmJsScriptFunction* asmJsFunc = AsmJsScriptFunction::FromVar(func);
```

Comment:
>func [](start = 70, length = 4)

func is guaranteed to be AsmJsFunction?

Reply:
No, this function is naive, but we don't ship templatized jit at this time and it needs serious rework if we're gonna revive it.